lib/audio/audio_controller.dart

```diff
@@ -111,6 +111,12 @@ class AudioController {
       await _soloud.play(_enVoice.startToPlay, volume: voiceVolume);
   Future<void> pass() async =>
       await _soloud.play(_enVoice.pass, volume: voiceVolume);
-  Future<void> count(int i) async =>
-      await _soloud.play(_enVoice.count[i - 1], volume: voiceVolume);
+  Future<void> count(int i) async {
```

Comment:
these changes caused TimeDisplay to send in `0`.  Should probably fix that interaction, but also good to fail gracefully rather than crashing.